ui/raidboss/data/03-hw/alliance/dun_scaith.js

```diff
@@ -164,13 +183,16 @@
     {
       id: 'Dun Scaith Blackfire',
       regex: / 14:1CAA:Ferdiad Hollow starts using Blackfire/,
+      regexDe: / 14:1CAA:Nihil-Ferdiad starts using Schwarzfeuer/,
+      regexFr: / 14:1CAA:Ferdiad Nihil starts using Feu Noir/,
+      regexJa: / 14:1CAA:フェルディア・ホロー starts using ブラックファイア/,
       infoText: {
         en: 'Avoid puddles',
       },
     },
     {
       id: 'Dun Scaith Debilitator',
-      regex: / 1A:\y{ObjectId}:\y{Name} gains the effect of (Fire|Water) Resistance Down/,
+      regex: / 1A:\y{ObjectId}:\y{Name} gains the effect of (Fire Resistance Down|Water Resistance Down)/,
```

Comment:
The `matches` for the alert text here will need to be updated as well.

Reply:
Matched condition updated

Reply:
Update on this one:
this one will need to be split OR need heavy support from Fr/Ja/Cn/Ko users.
Reason is: When adding other languages, the mathing term needs updates as well and i can only cover english and german

Reply:
It might not be necessary. The effects are available in XIVAPI and Saint Coinach. (Fire Resistance Down, for instance, is `status` [621](https://xivapi.com/status/621)). If that doesn't work, it might be possible to check against the "your resistance to water/fire plummets" message, at `logmessage` [2788](https://xivapi.com/logmessage/2788) and [2789](https://xivapi.com/logmessage/2789), although that wouldn't be as easy to automate due to the formatting on these two lines.

Reply:
Oh, I see what you're saying, the warning message. I'll try to run Dun Scaith tonight and see whether there's anything in the log we could use.

Reply:
Unfortunately it doesn't look like there is anything usable in the log. I didn't do a full network data parse, so that might have some info, but for now I think we'll be fine just localizing the regex and hoping for support on translating the actual warning.

Reply:
I will run this today to see whats the correct german names so we have at least an ID to work with here

Reply:
Could you please verify in your logs if it says "Fire/Water Resistance Down II" ?

Reply:
The code as written should work, considering the II will come after and not be captured. Although, it would be nice to have the correct line.

Reply:
Ah, you're right, it *is* Resistance Down 2. [1137 Fire](https://xivapi.com/status/1137) and [1157 Water](https://xivapi.com/status/1157).

Reply:
ok than i can quikly add language regexes and matches for it. Thank you

Reply:
ok now added needed regexes and matches. NOTE: The japanese Water Resistance Down II looks like it has a space at the end but i confirmed with my data and xivapi that it looks like this: "水属性耐性低下［強］". Same goes for the square openening bracket, which is not the standard one (but this only is valid for water and not for fire)

Reply:
Wait, one of them is fullwidth right/left square brackets and the other is just normall right/left square brackets??

Reply:
Yes, dont ask me why... i double checked both values from my data and xivapi.
Here is a direct copy paste: 
Fire: "火属性耐性低下[強]"
Water: "水属性耐性低下［強］"

Reply:
It does appear that way. The raw responses from XIVAPI:

Fire: `\u706b\u5c5e\u6027\u8010\u6027\u4f4e\u4e0b[\u5f37]`
Water: `\u6c34\u5c5e\u6027\u8010\u6027\u4f4e\u4e0b\uff3b\u5f37\uff3d`